src/style.css

```diff
@@ -0,0 +1,50 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: 50%;
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+}
+
+.stars--0 :first-child,
+.stars--1 :first-child,
+.stars--2 :first-child,
+.stars--3 :first-child,
+.stars--4 :first-child,
+.stars--5 :first-child {
+  margin-left: 0;
+}
```

Comment:
Maybe it would be better to use `stars__star:last_child` :)

Reply:
:last-child causes this
![image](https://user-images.githubusercontent.com/55781950/224402909-598f09e2-4e84-443b-b54b-2cf6de5f3f0e.png)

This is because the margin is on the left
:first-child produces expected results, thank you